Add new classes and it's first work version

Bjjyx opened the pull request:
Первая версия. 
При вводе числовых символов пользователем, дабы избежать ошибок, я использовал String.matches, что предполагает использование регулярных выражений, однако мы их не проходили. Отсюда вопрос, как в таком случае обойтись без регулярных выражений?

Bjjyx committed:
Add new classes and it's first work version

Charmandik commented on src/main/java/MainMenu.java:
```diff
@@ -0,0 +1,85 @@
+
+//            int peopleCount = scanner.nextInt();
+            String peopleCountStr = scanner.next();
+            if (peopleCountStr.matches("\\d*")){
```
вместо регулярок можно использовать конструкцию try/catch вместе с функцией nextInt(). Код будет ожидать число и если это будет не оно, то код будет падать с ошибкой, которую ты будешь ловить в catch.
Либо же сканер может проверять ввод с помощью функции hasNextInt(), которая возвращает true, если ввод может быть приведен к числу

Charmandik commented on src/main/java/MainMenu.java:
```diff
@@ -0,0 +1,85 @@
+    private void addProduct(){
+
+        System.out.println("Введите название товара");
+        String name = scanner.next();
```
метод next() получает только первое слово, если вводить например "вода минеральная", используй вместо nextLine()

Charmandik commented on src/main/java/MainMenu.java:
```diff
@@ -0,0 +1,85 @@
+            System.out.printf((message) + "%n", cheque, "рублей");
+        } else if (cheque >= 1.00 && cheque < 2.00) {
+            System.out.printf((message) + "%n", cheque, "рубль");
+        } else if (cheque >= 2.00 && cheque < 10.00) {
```
пять рубля, шесть семь восемь?

Bjjyx committed:
Исправлена ошибка при выводе цены от 5 до 10 рублей.
Изменена система проверки ввода из консоли.
Убран общий метод addProduct, на его месте теперь два отдельных метода addProductName и addProductCost.
В каждом методе теперь свой сканер, связанно это с тем, что общий сканер для всего класса MainMenu выдавал сбои.

Fixed an error when displaying prices from 5 to 10 rubles.
The system for checking input from the console has been changed.
The general method addProduct has been separate into two methods addProductName and addProductCost, and removed.
Each method now has its own scanner, because common scanner can makes errors.

Bjjyx committed:
Переработан метод showChequeAndProducts в классе MainMenu. Ошибка в падежах слова "рубль" исправлена.

Redesigned method showChequeAndProducts in the MainMenu class. The error in the cases of the word "ruble" has been corrected.

Bjjyx committed:
Добавлен новый метод getRubCase, в который передает слово "рубль" в падеже в зависимости от цены. Сделано это для более легкого чтения кода.

A new method getRubCase has been added, which transmits the word "ruble" in the case depending on the price. This is done for easier code reading.
